public/raven.min.js

```diff
@@ -0,0 +1,3 @@
+//# sourceMappingURL=raven.min.js.map
```

Comment:
@dannycoates Do we want to install [**raven**](https://www.npmjs.com/package/raven) via npm and use some script to copy the raven.min.js file into _/public/*_? Adding vendor code to repos always makes me uneasy.